### PR TITLE
Enh#11

### PR DESCRIPTION
**Pull request by henryk69**
# Enhancements

Updated whole page - should be fully featured now. Please check.

Missing: 
1. missing information should be filled (shown as FIXME) - mainly emails and phone no. No separate issue needed.
2. missing photos - see issue #28 
3. Links to homepages should be fixed - they should link to real empty pages for each person to fill #29

### CODE REVIEW

**Review by dizak — CHANGES_REQUESTED** [review]
There should be no style attributes in the HTML here. This is reverting the clean-up I did, you must have overwritten them.

File: people.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ sidebar_link: true
 ##### Omics data analysis and visualization
 <div class="flex-container">
   <img src="https://placehold.it/100x100" alt="">
-  <ul style="list-style-type:none;">
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 2]
```diff
@@ -16,7 +16,6 @@ sidebar_link: true
...
     <li><p style="margin-bottom: 0px"><a href="http://webjeda.com/online-cv/">Agnieszka Gromadka, MSc</a></p></li>
```
Comment (dizak) [inline people.md]: You should remove ```style``` attribute from the paragraph, like so:

```html
<li><p><a href="http://webjeda.com/online-cv/">Agnieszka Gromadka, MSc</a></p></li>
```
It applies to all ```<li>``` tags below

File: people.md [hunk 3]
```diff
@@ -31,7 +30,6 @@ sidebar_link: true
 
 <div class="flex-container">
   <img src="https://placehold.it/100x100" alt="">
-  <ul style="list-style-type:none;">
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 4]
```diff
@@ -42,7 +40,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 5]
```diff
@@ -53,7 +50,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 6]
```diff
@@ -64,7 +60,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 7]
```diff
@@ -73,7 +68,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 8]
```diff
@@ -82,7 +76,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 9]
```diff
@@ -91,7 +84,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

File: people.md [hunk 10]
```diff
@@ -100,7 +92,6 @@ sidebar_link: true
...
```
Comment (dizak) [inline people.md]: You should NOT remove the whole opening tag, just ```style``` attribute:
```html
<ul>
```

**Review by dizak — CHANGES_REQUESTED** [review]
You should remove the attributes NOT the whole tags

**Review by dizak — APPROVED** [review]
OK!

**Comment by henryk69** [comment]
this is so cool - I love merging changes!

**Pull request merged**